data/en/test_date_range_period_validation.json

```diff
@@ -0,0 +1,53 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "023",
+    "title": "Date formats",
+    "description": "A test schema for different date formats",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "variables": {
+        "period": "{{ format_conditional_date (answers.period_from, exercise.start_date)}} to {{ format_conditional_date (answers.period_to, exercise.end_date)}}"
+    },
+    "sections": [{
+        "id": "default-section",
+        "groups": [{
+            "id": "dates",
+            "title": "Date Range Validation",
+            "blocks": [{
+                    "type": "Question",
+                    "id": "date-range-block",
+                    "title": "Date Range",
+                    "questions": [{
+                            "id": "date-range-question",
+                            "title": "Enter Date Range",
+                            "type": "DateRange",
+                            "period_limits": {
+                              "min_value": 23,
```

Comment:
23 what? Years, Months, Days, Minutes, Seconds, Flamingos? Check the format of offset on date routing.

Reply:
Also does min_value define what this is? Is min_range clearer?

Reply:
its days, and min_limit perhaps ?

Reply:
ok so i think `min_day_limit` will clear it up a little ?

Reply:
We shouldn't limit ourselves to just days.

Reply:
This is a bit like the average stuff in the calculated validation, should i add the ability for it when there are no business requirements for it? And then a few days down the line im backing the changes out because its not needed. The card only asks for period limits to be set in days. I am more than happy to add the ability for other units of time but if it is reuested

Reply:
Consider using dateutil.relativedelta in your date comparison to help compare difference in years, months, days etc.